tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html

```diff
@@ -0,0 +1,18 @@
+<div class="grid__footer-contact footer-cta">
+    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading=contact_heading heading_level=2 classes="motif-heading--two motif-heading--static footer-cta__heading" %}
+
+    <p class="footer-cta__text">{{ contact_text }} </p>
+
+    <div class="footer-cta__grid">
+        {% include "patterns/atoms/avatar/avatar.html" with classes="footer-cta__avatar avatar--smallest" %}
+
+        <p class="footer-cta__cite">
+            <span class="footer-cta__name">{{ contact_name }}</span>
+            <span class="footer-cta__role">{{ contact_role }}</span>
+        </p>
+
+        <div class="footer-cta__button-wrapper">
+            <a href="contact_link" class="button footer-cta__button">Get in touch</a>
```

Comment:
Should `contact_link ` be a variable?

Reply:
fixed in 88e320342b19f7164c5148a950419e8012b98129